### PR TITLE
Multi action and parameter saving

### PR DESCRIPTION
**Pull request by sangbin-sangbin**

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -137,7 +175,7 @@ def normalize_points(points):
 
 wake_up_state = []
 
-visual_notification = ['', 0]
+is_multi_action = False
```
Comment (kmsgm) [inline test.py:178]: I thinks this variable is not used.

**Comment by sangbin-sangbin** [comment]
> LGTM, but plz describe the task you've done. What's the multi action for?
After finish one action, if gesture does not change for enough time, repeat action at certain amount of time intervals

**Comment by kmsgm** [comment]
> > LGTM, but plz describe the task you've done. What's the multi action for?
> > After finish one action, if gesture does not change for enough time, repeat action at certain amount of time intervals

Sorry I don't get it. So it's kinda resetting the gesture state?

**Comment by sangbin-sangbin** [comment]
> > > LGTM, but plz describe the task you've done. What's the multi action for?
> > > After finish one action, if gesture does not change for enough time, repeat action at certain amount of time intervals
> 
> Sorry I don't get it. So it's kinda resetting the gesture state?

제스처 유지하고있으면 연속으로 동작하는거예요. 왼쪽 계속 하고있으면 왼쪽으로 다라락 가는 그런거.

시간 있으시면 팀즈에 ppt도 보냈는데 봐주세요

File: test.py (hunk 1 above)
Comment (sangbin-sangbin) [reply test.py]: fixed

**Comment by kmsgm** [comment]
+) I think there is an issue regarding closing. `q` does not close the window until I answer the question.
It would be better first, close the window and then show the question. What do you think?
```
want to save current parameters? [ y / n ]
```

**Comment by sangbin-sangbin** [comment]
> +) I think there is an issue regarding closing. `q` does not close the window until I answer the question. It would be better first, close the window and then show the question. What do you think?
> 
> ```
> want to save current parameters? [ y / n ]
> ```

fixed

**Comment by kmsgm** [comment]
> > +) I think there is an issue regarding closing. `q` does not close the window until I answer the question. It would be better first, close the window and then show the question. What do you think?
> > ```
> > want to save current parameters? [ y / n ]
> > ```
> 
> fixed

The question shows up as soon as the screen open. Is this the intended way? Should it be poped up after enter the `q` button?

**Comment by sangbin-sangbin** [comment]
> > > +) I think there is an issue regarding closing. `q` does not close the window until I answer the question. It would be better first, close the window and then show the question. What do you think?
> > > ```
> > > want to save current parameters? [ y / n ]
> > > ```
> > 
> > 
> > fixed
> 
> The question shows up as soon as the screen open. Is this the intended way? Should it be poped up after enter the `q` button?

not in my laptop

**Comment by kmsgm** [comment]
> > > > +) I think there is an issue regarding closing. `q` does not close the window until I answer the question. It would be better first, close the window and then show the question. What do you think?
> > > > ```
> > > > want to save current parameters? [ y / n ]
> > > > ```
> > > 
> > > 
> > > fixed
> > 
> > 
> > The question shows up as soon as the screen open. Is this the intended way? Should it be poped up after enter the `q` button?
> 
> not in my laptop

Sry I was confused. Everything's good.

**Pull request merged**